jsapp/js/formbuild/containers/KoboMatrix.es6

```diff
@@ -176,15 +265,13 @@ class KoboMatrix extends React.Component {
         title: t('Change column type?'),
         message: t('Are you sure you want to change the type? This action is irreversible, your existing option choices will be erased.'),
         labels: {ok: t('Change type'), cancel: t('Cancel')},
-        onok: (evt, val) => {
+        onok: () => {
           data = data.setIn([colKuid, 'type'], newType);
           data = data.deleteIn([colKuid, 'select_from_list_name']);
           _this.setState({data: data});
           _this.toLocalStorage(data);
         },
-        oncancel: () => {
-          dialog.destroy();
-        }
+        oncancel: dialog.destroy,
```

Comment:
A naive question, but is there is a reason for this not to be `onOk` and `onCancel`?

Reply:
Oh, this thing is coming from a plugin -_- can't really do anything with it 🤷